<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>960</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="Global_Button">
            <property name="font">
             <font>
              <family>微軟正黑體</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>全球</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Asia_Button">
            <property name="font">
             <font>
              <family>微軟正黑體</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>亞洲</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="America_Button">
            <property name="font">
             <font>
              <family>微軟正黑體</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>美洲</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Europe_Button">
            <property name="font">
             <font>
              <family>微軟正黑體</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>歐洲</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Taiwan_Button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>微軟正黑體</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>台灣</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="National_debt_Button">
            <property name="font">
             <font>
              <family>微軟正黑體</family>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>國債</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Virtual_currency_Button">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>虛擬貨幣</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="autoScroll">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
